rebar.config

```diff
@@ -1,8 +1,12 @@
-{require_otp_vsn, "R15|R16|17"}.
+{require_otp_vsn, "R15|R16|17|18"}.
 
 {cover_enabled, true}.
 {eunit_opts, [verbose]}.
-{erl_opts, [warnings_as_errors, debug_info, nowarn_deprecated_type]}.
```

Comment:
Which warnings required removing `warnings_as_errors`? I've made a note of  basho/erlang_protobuffs#82 being a dependency of this PR.